Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/PaintBrushManipulator.cpp

```diff
@@ -343,13 +347,106 @@ namespace AzToolsFramework
                 .SetTip(PaintbrushDecreaseHardnessDesc)
                 .SetEntityComponentIdPair(m_ownerEntityComponentId)
                 .SetCallback(
-                    [this]()
+                    []()
                     {
                         AdjustHardnessPercent(-ed_paintBrushHardnessPercentAdjustAmount);
                     }),
         };
     }
 
+    void PaintBrushManipulator::RegisterActions()
```

Comment:
Please add some comments explaining why this exists and how it's different from PopulateActionsImpl(). It's very confusing from looking at the code why we have two different but similar methods that are dealing with Actions.